### ISSUE TITLE
Remove `ClearPreviousSBRP` target from `repo-projects/source-build-reference-packages.proj`

### ISSUE THREAD

**Issue by NikolaMilosavljevic** [dotnet/source-build#4210]
Publishing changes in `arcade` (https://github.com/dotnet/arcade/pull/14559) are replacing old model of propagating artifacts from repo build to shared VMR locations.

Remove `ClearPreviousSBRP` target once the arcade changes have flown to `installer` repo.

New publishing model also adds a new task for clearing previous SBRPs during inner-build - after building and before publishing.

**Comment by NikolaMilosavljevic** [comment]
Blocked by https://github.com/dotnet/installer/pull/19145
